test/functional/config_templates_controller_test.rb

```diff
@@ -79,7 +79,7 @@ class ConfigTemplatesControllerTest < ActionController::TestCase
       File.read(File.expand_path(File.dirname(__FILE__) + "/../../app/views/unattended/pxe/PXELinux_default.erb")))
     Setting[:unattended_url] = "http://foreman.unattended.url"
 
-    ProxyAPI::TFTP.any_instance.expects(:create_default).with(has_entry(:menu, regexp_matches(/http:\/\/foreman.unattended.url/))).returns(true)
+    ProxyAPI::TFTP.any_instance.expects(:create_default).with(has_entry(:menu, regexp_matches(/ks=http:\/\/foreman.unattended.url/))).returns(true)
```

Comment:
For completeness, can we even stronger regexp here? This is how things render in my case

    append initrd=boot/CentOS-6.5-x86_64-initrd.img ks=http://192.168.122.1/unattended/template/Kickstart%20default/with-root-pw ksdevice=bootif network kssendmac

so we can match for

    /ks=http:\/\/foreman.unattended.url\/template\//